src/v2/Apps/Auction/Routes/__tests__/Register.jest.tsx

```diff
@@ -237,4 +233,44 @@ describe("Routes/Register ", () => {
 
     expect(page.text()).toMatch("Telephone is required")
   })
+
+  it("displays an error message as the user types invalid input", async () => {
+    const env = setupTestEnv()
+    const page = await env.buildPage()
+
+    page
+      .find(CreditCardInput)
+      .props()
+      .onChange({ error: { message: "Your card number is invalid." } } as any)
+
+    expect(page.text()).toContain("Your card number is invalid.")
+  })
+
+  it("displays an error message when the input in the credit card field is missing", async () => {
+    const env = setupTestEnv()
+    const page = await env.buildPage()
+
+    createTokenMock.mockResolvedValue({
+      error: { message: "Your card number is incomplete." },
+    })
+
+    await page.fillFormWithValidValues()
+    await page.submitForm()
+
+    expect(page.text()).toContain("Your card number is incomplete.")
+  })
+
+  it("displays an error message when Stripe Element throws an error", async () => {
```

Comment:
to make sure I understand, this would be displaying an error message at the bottom of the page (like the bid+register form), instead of the screen-overlayed "An error occurred
Something went wrong. Please try again or contact support@artsy.net." modal, is that correct? If so, makes sense to me, just wanted to confirm.

Reply:
Correct. You can see that in the screenshot.